tests/components/viewControls-test.js

```diff
@@ -0,0 +1,112 @@
+jest.dontMock('../../js/components/viewControls')
+.dontMock('../../js/components/directionControl')
+.dontMock('../../js/components/utils')
+.dontMock('../../js/components/icon')
+.dontMock('../../js/components/logo')
+.dontMock('../../js/constants/constants')
+.dontMock('classnames');
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+var TestUtils = require('react-addons-test-utils');
+var skinConfig = require('../../config/skin.json');
+var CONSTANTS = require('../../js/constants/constants');
+var ViewControls = require('../../js/components/viewControls');
+var DirectionControl = require('../../js/components/directionControl');
+
+describe('viewControls', function () {
+  
+  var baseMockController, baseMockProps;
+  var defaultSkinConfig = JSON.parse(JSON.stringify(skinConfig));
+  
+  beforeEach(function () {
+    baseMockProps = {
+      isLiveStream: false,
+      controller: baseMockController,
+      skinConfig: JSON.parse(JSON.stringify(defaultSkinConfig))
+    };
+  });
+  
+  it('creates a viewControls', function () {
+    var controller = {
+      getVRParams: function () {
+        return {stereo: false};
+      }
+    };
+    
+    var mockProps = {
+      isLiveStream: false,
+      duration: 30,
+      skinConfig: skinConfig,
+      playerState: CONSTANTS.STATE.PLAYING,
+      controller: controller
+    };
+    
+    var DOM = TestUtils.renderIntoDocument(	<ViewControls {...mockProps}/> );
+  });
+  
+  it('create buttons in a viewControls', function () {
+    var mockProps = {
```

Comment:
Can we use `baseMockProps` in these tests instead of defining `mockProps` each time? Since `baseMockProps` gets recreated every time you can just set any custom values on each test without having to repeat the values that don't change.

Reply:
Yes. Thank you. We fix this in userstory 1424